### ISSUE TITLE
There are not relevant messages about containers after ./Zebrunner. sh restore

### ISSUE THREAD

**Issue by dhreben** [zebrunner/mcloud-agent#66]
**Steps:**

1../zebrunner.sh backup
2./zebrunner.sh shutdown
3../zebrunner.sh restore 

**Actual result :**  There are unnecessary messages about containers after ./Zebrunner. sh restore

```
./zebrunner.sh restore**
      Your services will be stopped and current data might be lost. Do you want to do a restore now? y/n [n]:y
sudo: unable to resolve host mcloud
sudo: unable to resolve host mcloud
sudo: unable to resolve host mcloud
sudo: unable to resolve host mcloud
MCloud restore succeed.
removing device-Nokia_3-....... containers...
Error: No such container: device-Nokia_3-......
Error: No such container: device-Nokia_3.....-appium
removing device-Samsung_Galaxy_S10-....... containers...
Error: No such container: device-Samsung_Galaxy_S10-R.......
Error: No such container: device-Samsung_Galaxy_S10-R......-appium
removing device-Samsung_S5-4..... containers...
Error: No such container: device-Samsung_S5-4.....
Error: No such container: device-Samsung_S5-4d.....-appium


      WARNING! Your services needs to be started after restore.
      Start now? y/n [y]:y
```

**Comment by vdelendik** [comment]
fixed in scope of #75

**Comment by dhreben** [comment]
Verified.
